src/Common/Uuid/UuidRegistry.php

```diff
@@ -183,4 +183,33 @@ public static function uuidToBytes($uuidString)
     {
         return Uuid::fromString($uuidString)->getBytes();
     }
+
+    /**
+     * Check if UUID String is Valid
+     * @return boolean
+     */
+    public static function isValidStringUUID($uuidString)
+    {
+        return (UUID::isValid($uuidString));
+    }
+
+    /**
+     * Check if UUID String is Empty
+     * @return boolean
+     */
+    public static function isEmptyStringUUID($uuidString)
+    {
+        return (!empty($uuidString) &&
+            $uuidString != null &&
+            $uuidString != '');
+    }
+
+    /**
+     * Check if UUID Brinary is Empty
+     * @return boolean
+     */
+    public static function isEmptyBinaryUUID($uuidString)
```

Comment:
plus I think for both of these functions, now need to reverse the comparisons since now checking if empty rather than if valid (or not empty)